### ISSUE TITLE
no information populating

### ISSUE THREAD

**Issue by danmartinez78** [shevabam/ezservermonitor-web#39]
I'm running a raspian jessie on a raspi 2 B+. I placed the ezserver files in /var/www/html, with ownership set to root:root. index.php loads when I go to the raspi's ip address but no information is populated in the displays. Any ideas?

**Comment by shevabam** [comment]
Hi,

- Can you give me a screenshot?
- Which version of PHP is installed?
- Apache / nginx ? Which version ?

**Comment by danmartinez78** [comment]
![image](https://cloud.githubusercontent.com/assets/11601571/17969746/3a53aec8-6a99-11e6-81ff-02b7aec5be60.png)

PHP 5.6.24-0+deb8u1 (cli) (built: Jul 27 2016 00:06:04)
Server version: Apache/2.4.10 (Raspbian)

**Comment by maarek** [comment]
I see this same issue on mine. 

PHP 5.6.20-0+deb8u1 (cli) (built: Apr 28 2016 00:01:26)
lighttpd/1.4.35 (ssl) (Nov  7 2014 12:24:03) - a light and fast webserver

**Comment by shevabam** [comment]
Is there something wrong in the Apache/Lighttpd logs?

**Comment by rnikhil275** [comment]
Even I have the same issue.

**Comment by ravenclaw69** [comment]
Hi, i have the same issue with nginx 1.6.2 / PHP 5.6.24-0+deb8u1 in Debian Jessie 8.0 !
I'm run esm for 2 / 3 hours, but when i came back esm display like @danmartinez78. 

I try to restart / reload / stop / start PHP, nginx and my rasp pi 3 and nothing ^^
I have nothing in my log too.

Thanks for you support.

**Comment by kernalzero** [comment]
![ezserver_blank](https://cloud.githubusercontent.com/assets/11688574/22407341/a5c3cdb8-e632-11e6-87d3-0e8c5b1f6413.jpg)

I'm having the same issue on a RPi B
Jessie Lite installed 
4.4.38+ GNU/Linux

Lighttpd
lighttpd/1.4.35 (ssl) - a light and fast webserver
Build-Date: Aug  6 2016 04:23:34

PHP - 
PHP 5.6.29-0+deb8u1 (cli) (built: Dec 17 2016 06:04:43) 
Copyright (c) 1997-2016 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.0.6-dev, Copyright (c) 1999-2016, by Zend Technologies

Has anyone resolved this issue?  I'm wanting to use this to monitor the status of my pihole DNS Server.  Thanks

**Comment by shevabam** [comment]
Hi

In your php.ini, do you have the functions `exec` or `shell_exec` in `disable_functions` directive?

**Comment by kernalzero** [comment]
I search through my php.ini file and I don't see either.

**Comment by Asenar** [comment]
Hi,

you can try to run one of the scripts in libs/ from the command line

**Comment by MatthewSH** [comment]
Hey @shevabam. I am getting the same issue. PHP 7.1, Ubuntu 16.04.02 LTS. Nginx 1.11.9

It seems it has to do with the following notice showing up:
system.php
```

Notice: A non well formed numeric value encountered in /***/libs/Utils/Misc.php on line 92

Notice: A non well formed numeric value encountered in /***/libs/Utils/Misc.php on line 92

Notice: A non well formed numeric value encountered in /***/libs/Utils/Misc.php on line 101
```

And the list goes on for it on every file. You're assuming (based on https://github.com/shevabam/ezservermonitor-web/blob/master/js/esm.js#L326 ) that everything is coming back properly as JSON, but it's not. Those notices actually prohibit the modules from displaying.

To fix the `system.php` I had to cast 
```php 
$seconds = (int) $seconds
```
Reference: https://github.com/shevabam/ezservermonitor-web/blob/master/libs/Utils/Misc.php#L78 

Or you could just add `error_reporting(0);` to the top of `load_average.php`, `swap.php`, `memory.php`, and `system.php`. However, I wouldn't suggest this as the notice is showing that something is messed up. 

Another side note is that my load averages are always sitting at 0%, but that's just because for some reason `/proc/loadavg` is returning 0. I'll figure that out at a later time though.

**Comment by shevabam** [comment]
@MatthewSH Thanks a lot for your analysis! I fixed PHP notices 725787bcf7421143d6567e5b4f39ab5fd3fd8695

**Comment by MatthewSH** [comment]
@shevebam no problem! Glad you could get it fixed!
